.appveyor.yml

```diff
@@ -96,14 +91,13 @@ test_script:
   - cd ..
 
 on_success:
-  # Upload the generated wheel package to Rackspace
-  # On Windows, Apache Libcloud cannot find a standard CA cert bundle so we
-  # disable the ssl checks.
-  - pip install wheelhouse-uploader
-  - python -m wheelhouse_uploader upload
-    --local-folder=%REPO_DIR%\dist
-    %UPLOAD_ARGS%
-    %CONTAINER%
+  # Upload the generated wheel package to anaconda.org
+  # PYWAVELETS_STAGING_UPLOAD_TOKEN is an encrypted variable
+  # used in Appveyor CI config, originally created at
+  # multibuild-wheels-staging site
+  - cd ..\scikit-image
```

Comment:
Appveyor is failing here with a "cannot find the path specified". 

https://ci.appveyor.com/project/scikit-image/scikit-image-wheels/builds/35551521/job/rc46igsp33tfmnu2#L2376